package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_most_searched_website_on-google",
+  "version": "1.0.0",
+  "description": "Idea title : Most googled website in 2023",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
The "test" script currently just echoes an error message and exits. It would be more useful to have this script actually run some tests. If there are no tests to run yet, consider at least setting up a testing framework. [important]